### PR TITLE
[COD-17]

### PR DESCRIPTION
**Pull request by oksygenn**

### CODE REVIEW

File: components/SiteHeader/Dropdown.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        onClick={() => setIsHidden(!isHidden)}
+        className="hover:text-blue-800"
+      >
+        Categories
+      </button>
+      <div
+        className={
+          isHidden
+            ? "dropdown-content hidden absolute w-96 bg-gray-50 z-10 shadow p-8 my-8 border border-gray-200 rounded"
+            : "dropdown-content block absolute w-96 bg-gray-50 z-10 shadow p-8 my-8 border border-gray-200 rounded"
```
Comment (artcodespace) [inline components/SiteHeader/Dropdown.js]: Both of the outcomes in this ternary are _very_ similar. The only difference seems to be that one has `hidden`, and one has `block`. Can you think of a way of writing this without writing all of the other classes twice?

File: pages/index.js [hunk 2]
```diff
@@ -21,7 +23,7 @@ const Index = () => {
         imgUrl="https://ichef.bbci.co.uk/childrens-responsive-ichef-ck/400xn/amz/cbbc/bp-duckling.jpg"
         imgAltText="duckling"
         buttonText="Learn more about Miricyl's vision"
-        buttonLink="/about"
+        buttonLink="/mental-health"
```
Comment (artcodespace) [inline pages/index.js:26]: What is this prop being used for? It looks to me like this prop isn't used at all in the `Header` component

Comment (oksygenn) [reply pages/index.js:26]: It is used in CalloutBoxButton

File: components/SiteHeader/Dropdown.js (hunk 1 above)
Comment (oksygenn) [reply components/SiteHeader/Dropdown.js]: Done! Thanks. I didn't know there's is a better way

File: pages/index.js (hunk 2 above)
Comment (artcodespace) [reply pages/index.js:26]: Ha! Sorry, ignore me, I didn't spot I needed to expand the view on Github

**Review by artcodespace — APPROVED** [review]

**Pull request merged**